### DISCUSSION TITLE
Deployment Problems

### DISCUSSION THREAD

**Discussion by ishubhamsingh2e** [hmislk/hmis#6802] [Q&A]
I am trying to build the project and am having some problems in deploying Glassfish is throwing this error on deployment

jakarta.enterprise.system.core|_ThreadID=213;_ThreadName=admin-listener(6);_LevelValue=1000;|
  Exception while loading the app : java.lang.IllegalStateException: ContainerBase.addChild: start: org.apache.catalina.LifecycleException: java.lang.IllegalArgumentException: java.lang.IllegalArgumentException: Unable to add listener of type: org.primefaces.webapp.UploadedFileCleanerListener, because it does not implement any of the required ServletContextListener, ServletContextAttributeListener, ServletRequestListener, ServletRequestAttributeListener, HttpSessionListener, or HttpSessionAttributeListener interfaces|#]
  
  I tried building the main, Issue#6494, 6631, and 6784

**Comment by ramathmanjitha27** [comment]
Hi, 
Hope it was solved
